OpenRA.Mods.Common/Traits/RevealsMap.cs

```diff
@@ -36,13 +36,7 @@ public RevealsMap(RevealsMapInfo info)
 				: Shroud.SourceType.PassiveVisibility;
 		}
 
-		protected void AddCellsToPlayerShroud(Actor self, Player p, PPos[] uv)
-		{
-			if (!Info.ValidRelationships.HasRelationship(self.Owner.RelationshipWith(p)))
-				return;
-
-			p.Shroud.AddSource(this, type, uv);
-		}
+		protected void AddCellsToPlayerShroud(Actor self, Player p, PPos[] uv) { p.Shroud.AddSource(this, type, uv); }
```

Comment:
Oh, sorry I had not realized `AddCellsToPlayerShroud` existed and established the pattern of handling relationships internally. :(